package.json

```diff
@@ -19,7 +19,7 @@
         "lint": "eslint src/**/*.ts",
         "format": "eslint src/**/*.ts --fix",
         "build:dev": "tsup --metafile",
-        "build:prod": "tsup --minify",
```

Comment:
we can use terser for this maybe

Reply:
tsup comes built in with minify, no need to minify